contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+After=docker.service
+Requires=docker.service
+Before=origin-master.service
```

Comment:
If I'm running both master and node I see that it will be fine, but what if i'm not running master on this node?